### PR TITLE
Milestone #1: Finish mobile version

### PR DESCRIPTION
**Pull request by fezzopro**
## Description
This PR contains instructions from [Portfolio: Setup and Mobile First](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

## introduced changes

* Linter configuration files from [Microverse Linter](https://github.com/microverseinc/linters-config)  
* [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
* Portfolio Mobile First changes reflected on:
 * index.html
- [ ] Added project section
- [ ] Added about me section
- [ ] Added contact me section

 * style.css
 * README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,6 +51,200 @@ <h1>I’m KAMANA. Glad to see you!</h1>
...
+          </p>
+          <ul class="stacks">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="more">
+            <a href="#"
+              ><button type="button" class="btn-more">See Project</button></a
+            >
```
Comment (jerryowusu) [inline index.html:134]: As part of the project requirement, kindly make sure that you implement the button styling for Active, Hover and Pressed for this and all other buttons on the page. 😉

File: index.html [hunk 2]
```diff
@@ -49,6 +51,200 @@ <h1>I’m KAMANA. Glad to see you!</h1>
...
```
Comment (jerryowusu) [inline index.html:107]: As part of the project requirement, kindly make sure that you implement the button styling for Active, Hover and Pressed for this and all other buttons on the page. 😉

File: index.html [hunk 3]
```diff
@@ -49,6 +51,200 @@ <h1>I’m KAMANA. Glad to see you!</h1>
...
```
Comment (jerryowusu) [inline index.html:80]: As part of the project requirement, kindly make sure that you implement the button styling for Active, Hover and Pressed for this and all other buttons on the page. 😉

File: index.html [hunk 4]
```diff
@@ -49,6 +51,200 @@ <h1>I’m KAMANA. Glad to see you!</h1>
...
```
Comment (jerryowusu) [inline index.html:161]: As part of the project requirement, kindly make sure that you implement the button styling for Active, Hover and Pressed for this and all other buttons on the page. 😉

File: index.html [hunk 5]
```diff
@@ -49,6 +51,200 @@ <h1>I’m KAMANA. Glad to see you!</h1>
...
+          <li class="language-item">
+            <img src="assets/images/Icons/css.png" alt="CSS" />
+            <p>CSS</p>
+          </li>
+          <li class="border-bottom">
+            <p>Frameworks</p>
+            <p>></p>
+          </li>
+
+          <li class="border-bottom">
```
Comment (jerryowusu) [inline index.html:222]: The space between the images and the language text(JavaScript, HTML, CSS) is way bigger than that of the actual design as shown below. 👇. Kindly adjust it to match the design. 

### Your design 

Screenshot from 2023-03-24 21-07-05



### Actual design 



Screenshot from 2023-03-24 21-06-50

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @fezzopro , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:


so-close


## Highlights
No linter errors :+1: 
Great job so far :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @fezzopro  :wave: 
 
Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: Descriptive pull request.
:heavy_check_mark: Meaningful commit message.
:heavy_check_mark: Follow gitflow

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**